pyproject.toml

```diff
@@ -17,23 +17,25 @@ readme = "README.md"
 dev = [
   "ruff",
   "black",
-  "pre-commit"
+  "pre-commit",
+  "pytest",
+  "scipy",
```

Comment:
Same here. The rationale for sorting is so that it's easy to find exactly what you want at a glance instead of explicitly searching.